noxfile.py

```diff
@@ -89,9 +89,9 @@ def test(session: nox.Session) -> None:
         shutil.rmtree(sdist_dir, ignore_errors=True)
 
     # fmt: off
-    session.install("setuptools")
+    session.install("build")
     session.run(
-        "python", "setup.py", "sdist", "--formats=zip", "--dist-dir", sdist_dir,
```

Comment:
I wonder if there's a mysterious reason behind building an sdists and not a wheel... In Hynek's blog posts, there's a sentiments that testing with wheels is faster (in the context of tox, though).

Reply:
I traced to #7153 but couldn’t find any further context. Maybe @pradyunsg remembers.

My suspicion is because this was (is?) what Tox does, and the two ways to run tests should be as similar as possible.